packages/Uploader/src/components/File/File.js

```diff
@@ -100,16 +104,24 @@ function File(props) {
       case types.status.IDLE:
         return I18n.t("uploader.progress.idle");
       default:
-        return I18n.t("uploader.progress.uploading", { progressWithUnits, sizeWithUnits });
+        if (!showA11yProgress) return I18n.t("uploader.progress.uploading", { progressWithUnits, sizeWithUnits });
+        return I18n.t("uploader.progress.uploading_percent", { a11yProgress });
```

Comment:
```suggestion
        return showA11yProgress
          ? I18n.t("uploader.progress.uploading_percent", { a11yProgress })
          :  I18n.t("uploader.progress.uploading", { progressWithUnits, sizeWithUnits });
```

Reply:
You may need to use the `showA11yProgress` for some of the other statuses too to get the `aria-labels` to sound right.